css-two-column-layout/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+ <div class="container-all">
+    <div class="row1"></div>
+      <div class="column-full">
+                <h1>SCALE, SPEED, AGILITY</h1>
+                <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium</p>
+                <p>network run 10x for efficiently and deliver an unparallelled quality of care.</p></div>
+
+
+    <div class="box1">
+  <div class="row2">
+    <div class="img1">
+      <img src="images/software.svg">
+    </div>
+    <div class="img1">
+      <h3>Intuitive, <div>Smart Software</div></h3>
+      <p>We build products that transform how
+        healthcare facilities run across Africa
+        -making them radically more efficient
+        operationally and financially, a complete
+        turnaround.
+      </p>
+    </div>
+ </div>
+
+<div class="row2">
```

Comment:
You are moving backwards in indentation, but it is just a sibling to the element above.